src/ftest.jl

```diff
@@ -30,32 +30,39 @@ end
 """
     ftest(mod::LinPredModel...)
 
-For each sequential pair of linear predictors in `mod`, perform an F-test to determine if the first one fits significantly better than the second
+For each sequential pair of linear predictors in `mod`, perform an F-test to determine if 
+the first one fits significantly better than the second.
 
-Note: This function can easily be used to do an ANOVA. ANOVA is nothing more than a test
-to see if one model fits the data better than another.
+A table is returned containing residual degrees-of-freedom, difference in residual DOF
```

Comment:
The difference in residual DOF isn't returned now. Would also be nice to specify DOF and SSR in parentheses after spelling them in full.